lecture_01/hw/task01.md

```diff
@@ -0,0 +1,11 @@
+## Warming up
+
+1. please create your own git repository on github.
+1. setup pre-commit hook with `black` and `isort` formatting for the repo
```

Comment:
It is not a trivial task for Windows, I'd recommend using https://pre-commit.com/

Reply:
I'd recommend students to use base cli tools instead of this combine. I'll mention that in the slideshow. 

In case of windows they can use WSL